src/main/java/zmq/Ctx.java

```diff
@@ -127,6 +134,13 @@ private void destroy() throws IOException
             it.close();
         }
 
+        for (Selector selector : selectors) {
+            if (selector != null) {
```

Comment:
I've opted to add an `assert(selector != null)` upon opening a Selector, and also checking for null before closing them, just in case, since there is some possibility that calling code could be setting a selector to `null`.